frontend/src/app/overview/overview.component.ts

```diff
@@ -11,18 +11,44 @@ import { NotifierService } from '../shared/services/notifier.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class OverviewComponent implements OnInit {
-  overviewEntities!: Observable<OverviewEntity[]>;
+  overviewEntities: Subject<OverviewEntity[]> = new Subject<OverviewEntity[]>();
 
   constructor(
     private overviewService: OverviewService,
     private notifierService: NotifierService,
   ) {
     this.notifierService.reloadOverview.subscribe(() => {
-      this.overviewEntities = this.overviewService.getOverview();
+      this.loadOverview();
     });
   }
 
   ngOnInit(): void {
-    this.overviewEntities = this.overviewService.getOverview();
+    this.loadOverview();
+  }
+
+  loadOverview() {
```

Comment:
Use "activatedRoute" instead of window to get router data: #445 could help you (sidepanel.component.ts)